src/shared/components/case-list/case-list.component.ts

```diff
@@ -43,11 +43,11 @@ export class CaseListComponent {
   constructor(private browserService: BrowserService) { }
 
   public formatDate(date: Date): string {
-    return formatDate(date, 'dd MMM yyyy', 'en-GB');
+    return date ? formatDate(date, 'dd MMM yyyy', 'en-GB') : '-';
   }
 
   public formatDateAtTime(date: Date): string {
-      return DateTimeFormatUtils.formatDateAtTime(date, false);
+    return date ? DateTimeFormatUtils.formatDateAtTime(date, false) : '-';
```

Comment:
I don't think the unit tests are actually testing this one at all. Can you add one for it, please?

Reply:
Added unit test case for formatDateAtTime function